pkg/lib/cockpit-components-password.jsx

```diff
@@ -77,20 +82,34 @@ export const PasswordFormFields = ({
                         setPasswordMessage(strength.message);
                     });
         } else {
-            setPasswordStrength("");
+            setPasswordStrength();
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19220-20230918-120248-8fd6ef98-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)